packages/xray-toolset/PKGBUILD

```diff
@@ -0,0 +1,60 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=xray-toolset
+pkgver=EvilPot.0.0.2.r1.gaee9f9c
+pkgrel=1
+pkgdesc='A complete security assessment tool that supports common web security issues scanning and custom POC.'
+arch=('x86_64')
+groups=('blackarch' 'blackarch-scanner' 'blackarch-webapp')
+url='https://github.com/chaitin/xray'
+license=('custom:unknown')
+depends=()
+makedepends=('git')
+source=("git+https://github.com/chaitin/xray.git"
+        "git+https://github.com/chaitin/xray-plugins.git"
+        "https://github.com/chaitin/xray/releases/download/xpoc-0.1.0/xpoc_linux_amd64"
+        "https://github.com/chaitin/xray/releases/download/xapp-0.0.2/xapp_linux_amd64"
+        "https://github.com/chaitin/xray-plugins/releases/download/xlint-0.0.3/xlint_linux_amd64")
```

Comment:
add xray-plugins as dependency and built it https://github.com/chaitin/xray-plugins/